gulpfile.js

```diff
@@ -0,0 +1,56 @@
+const { src, dest, series } = require('gulp');
+const purgecss = require('gulp-purgecss')
+const htmlmin = require('gulp-htmlmin');
+const cleanCSS = require('gulp-clean-css');
+const concat = require('gulp-concat');
+const del = require('del');
+const cp = require("child_process");
+const revAll = require("gulp-rev-all");
+
+const clean = () => {
+  return del(['dist']);
+}
+
+const build = () => {
+  return cp.spawn("npx", ["@11ty/eleventy", "--quiet"], { stdio: "inherit" });
+}
+
+const minifyHTML = () => {
+  const options = {
+    removeAttributeQuotes: true,
+    removeComments: true,
+    collapseWhitespace: true
+  }
+
+  return src('dist/**/*.html')
+   .pipe(htmlmin({ collapseWhitespace: true }))
```

Comment:
I think it should be
```suggestion
   .pipe(htmlmin(options))
```